packages/appstore/src/constants/tour-steps-config.tsx

```diff
@@ -1,80 +1,111 @@
 import { Step, Styles, Locale } from 'react-joyride';
 import React from 'react';
-import { Text, SpanButton } from '@deriv/components';
-import { Localize, localize } from '@deriv/translations';
+import { Text, SpanButton, Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
 import 'Components/toggle-account-type/toggle-account-type.scss';
+import { isMobile } from '@deriv/shared';
 
-export const tour_step_config: Step[] = [
+export const getTourStepConfig = (): Step[] => [
     {
         title: (
             <React.Fragment>
                 <Text as='p' weight='bold' color='brand-red-coral'>
-                    {localize('Switch accounts')}
+                    <Localize i18n_default_text='Switch accounts' />
                 </Text>
                 <div className='toggle-account-type__divider' />
             </React.Fragment>
         ),
-        content: <Text as='p'>{localize('Switch between your demo and real accounts.')}</Text>,
-        target: '.toggle-account-type__button',
+        content: (
+            <Text as='p'>
+                <Localize i18n_default_text='You can switch between real and demo accounts.' />
+            </Text>
+        ),
+        target: '.account-type-dropdown--parent',
         disableBeacon: true,
-        placement: 'bottom-end',
+        disableOverlayClose: true,
     },
     {
         title: (
             <React.Fragment>
                 <Text as='p' weight='bold' color='brand-red-coral'>
-                    {localize('Trading hub tour')}
+                    <Localize i18n_default_text='Choice of regulation' />
                 </Text>
                 <div className='toggle-account-type__divider' />
             </React.Fragment>
         ),
         content: (
             <Text as='p'>
                 <Localize
-                    i18n_default_text='Need help moving around?<0></0>We have a short tutorial that might help. Hit Repeat tour to begin.'
-                    components={[<br key={0} />]}
+                    i18n_default_text='You can create real accounts under EU or non-EU regulation. Click the <0><0/> icon to learn more about these accounts.'
+                    components={[
+                        <Text key={0}>
+                            <Icon icon='IcInfoOutline' />
+                        </Text>,
+                    ]}
                 />
             </Text>
         ),
 
-        target: '.trading-hub-header__tradinghub--onboarding--logo',
+        target: isMobile() ? '.main-title-bar-mobile--regulator' : '.regulators-switcher__container',
```

Comment:
Instead of this please use is_mobile from ui-store :grimacing: 
isMobile() is deprecated https://deriv-group.slack.com/archives/C3AHMSM3P/p1692604107332739

Reply:
actually its a plain function, and therefore we can't use store here. could there be any other way of removing isMobile?

Reply:
I guess the changes would be too much :thinking: let's keep it like this for now, later we can replace that